modules/core/src/main/java/org/apache/ignite/mxbean/IgniteMXBean.java

```diff
@@ -506,6 +510,32 @@ void runIoTest(
     @MXBeanParametersDescriptions("Metrics registry.")
     public void resetMetrics(String registry);
 
+    /**
+     * Change {@link HitRateMetric} configuration.
+     * Call of this method will change metric configuration across all cluster nodes.
+     *
+     * @param name Metric name.
+     * @param rateTimeInterval New rate time interval.
+     * @throws IgniteException If some error occured.
+     */
+    @MXBeanDescription("Configure hitrate metric by name.")
+    @MXBeanParametersNames({"name", "cfg"})
+    @MXBeanParametersDescriptions({"Metric name.", "New rate time interval."})
+    public void configureHitRateMetric(String name, long rateTimeInterval) throws IgniteException;
+
+    /**
+     * Change {@link HistogramMetric} configuration.
+     * Call of this method will change metric configuration across all cluster nodes.
+     *
+     * @param name Metric name.
+     * @param bounds New bounds.
+     * @throws IgniteException If some error occured.
+     */
+    @MXBeanDescription("Configure histogram metric by name.")
+    @MXBeanParametersNames({"name", "cfg"})
+    @MXBeanParametersDescriptions({"Metric name.", "New bounds."})
+    public void configureHistogramMetric(String name, long[] bounds) throws IgniteException;
```

Comment:
I think we could add `configure` method to the beans that expose metrics from registries for each histogram metric on the fly. What do you think?

Reply:
`JMXExporter` should be explicitly configured.
I think we should create separate JMX bean for metrics configuration purposes.
It will be created independently with `JMXExporter` so metrics configuration will be always accessible.

Reply:
Sounds good. But needs some additional thinking. I think on this iteration separate been will be good. Something like `MetricsMXBean` which will contains management methods.

Reply:
`MetricsMXBean` introduced.